### PR TITLE
fix(config): Massage config everytime

### PR DESCRIPTION
**Pull request by l0nax**
The config has only been "massaged" when presets where used.
This lead into problems as described in #9639.

Ref #8763

After this PR has been merged we can undo #9641.

## Changes:

When the config is read (via `getConfig(…)`) it was returned directly, without calling `massageConfig(…)`.
`massageConfig(…)` is only called when presets are used in the `renovate.json` config.

## Context:

Used renovate config to test against real repos:
```js
const baseConfig = {
  semanticCommits: "disabled",
  // Configure to `false` to disable deleting orphan branches and autoclosing PRs.
  pruneStaleBranches: false,
  enabledManagers: [
    'dockerfile',
  ],
};

module.exports = {
  dryRun: true,
  autodiscover: true,
  autodiscoverFilter: 'rentest/**/*',
  dependencyDashboard: true,
  dependencyDashboardAutoclose: true,
  persistRepoData: true,
  endpoint: "https://********/api/v4/",
  logLevel: "debug",
  platform: "gitlab",
  printConfig: false,
};

```

The group structure in GitLab is as shown:
```
rentest/
├── nicegrp/
│   ├── prj-1
│   └── prj-2
```

Renovate processes `prj-1` and `prj-2`

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added unit tests, or
- [ ] No new tests but ran on a real repository, or
- [x] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/config/file.ts [hunk 1]
```diff
@@ -30,5 +31,5 @@ export function getConfig(env: NodeJS.ProcessEnv): GlobalConfig {
     );
     config = migratedConfig;
   }
-  return config;
+  return massageConfig(migratedConfig);
```
Comment (viceice) [inline lib/config/file.ts:34]: @rarkins I'm not sure if this is the right place. 🤔

Comment (l0nax) [reply lib/config/file.ts:34]: The alternative place would be here:
https://github.com/renovatebot/renovate/blob/89ad6401d931499fd253720bc7741a4afcba9d7a/lib/workers/repository/init/config.ts#L7-L9

Then the massage would also apply to the encrypted parts of the configuration.

Comment (viceice) [reply lib/config/file.ts:34]: We need to make sure it won't message twice, thats my concern. Do not have the time to check this yet. sorry.

**Comment by rarkins** [comment]
@l0nax I'm going to close this for now due to inactivity and conflicts, but if you'd still like to see it then can you raise an issue first to discuss the requirements along with reproduction steps?

**Pull request closed without merging**